voter/src/main/java/org/terracotta/voter/VetoVote.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ *  The contents of this file are subject to the Terracotta Public License Version
+ *  2.0 (the "License"); You may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *
+ *  http://terracotta.org/legal/terracotta-public-license.
+ *
+ *  Software distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+ *  the specific language governing rights and limitations under the License.
+ *
+ *  The Covered Software is Terracotta Core.
+ *
+ *  The Initial Developer of the Covered Software is
+ *  Terracotta, Inc., a Software AG company
+ *
+ */
+package org.terracotta.voter;
+
+import java.util.Arrays;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ */
+public class VetoVote {
```

Comment:
To be removed in the final PR.

Reply:
Are you opposed?  Useful to me as an internal tool.  Would like is accessible for integration tests.

Reply:
Oh I thought you just forgot to remove it after your testing. But if it was intentional then I'm fine with keeping it internally.

BTW, If it's intended to be used in integration tests as well then just wondering if the direct logback logger usage would clash with the logger impl used in integration tests. If they're the same the this should be fine.